pipeline/preprocessing/ground_truth_processing/GroundTruthHandler.py

```diff
@@ -0,0 +1,84 @@
+from io import TextIOWrapper
+import json
+import pandas as pd  # type: ignore
+import os
+
+
+class GroundTruthHandler:
+    """This class handles formatting the "dirty" ground truth dataset and
+    also handles merging the ground truth with the FCD dataset for use in model training
+    """
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def load_from_geojson(cls, file: TextIOWrapper, year: int):
+        """Loads the ground truth from the dirty geojson and cleans it
+
+        Args:
+            file (TextIOWrapper): the file of the ground truth geojson.
+            year (int): The year that the ground truth speed limit is from at the latest.
+
+        Returns:
+            _type_: A dataframe that has 2 coloumns: osm_id and KODE_HAST_GENEREL_HAST, the latter being the speed limit
+        """
+        data = json.load(file)
+        return cls.__clean_vejman_data(data, year)
+
+    @classmethod
+    def __clean_vejman_data(cls, df: pd.DataFrame, year: int) -> pd.DataFrame:
+        """Cleans the dataframe by removing the superflous coloumns along
+            with filtering speed limits that have been changed too recently.
```

Comment:
Cleans the dataframe by removing the unnecessary columns along
            with filtering, speed limits that have been changed too recently.